bindings/swift/Sources/OpenDAL/Data+OpenDAL.swift

```diff
@@ -0,0 +1,39 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import Foundation
+import COpenDAL
+
+extension Data {
```

Comment:
Is it required for us to introduce this extension?

For example, aws swift sdk supports using `String`:

```swift
/// Returns ByteStream from a String object.
/// - Parameter stringValue: String object to be converted to ByteStream.
/// - Returns: ByteStream representation of the String object.
public static func from(stringValue: String) -> ByteStream {
    return .data(stringValue.data(using: .utf8) ?? Data())
}
```

Reply:
The extension is private to this binding package and will not be exposed to users. It’s mainly for creating `Data` from `opendal_bytes` type, and this is preferred over a freestanding function.